db/create_tables.sql

```diff
@@ -39,7 +39,7 @@ CREATE TABLE IF NOT EXISTS `events` (
   `type` VARCHAR(100) NOT NULL,
   `status` VARCHAR(100) DEFAULT NULL,
   `state_machine_instance_id` VARCHAR(255) NOT NULL,
```

Comment:
state_machine table doesn't contain state machine instance details. It is to store only state metadata.